be/src/olap/memtable.cpp

```diff
@@ -243,11 +242,23 @@ void MemTable::_collect_vskiplist_results() {
     VecTable::Iterator it(_vec_skip_list.get());
     vectorized::Block in_block = _input_mutable_block.to_block();
     if (_keys_type == KeysType::DUP_KEYS) {
+        vectorized::MutableBlock mutable_block =
+                vectorized::MutableBlock::build_mutable_block(&in_block);
+        _vec_row_comparator->set_block(&mutable_block);
+        std::sort(_row_in_blocks.begin(), _row_in_blocks.end(),
```

Comment:
Can we use `std::stable_sort` here directly? 
 `std::stable_sort(_row_in_blocks.begin(), _row_in_blocks.end(), _vec_row_comparator);`